kinto/__main__.py

```diff
@@ -98,16 +118,18 @@ def main(args=None):
                 import pip
                 pip.main(['install', "kinto[postgresql]"])
 
-    elif parsed_args['which'] == 'migrate':
+    elif which_command == 'migrate':
         dry_run = parsed_args['dry_run']
-        try:
-            logging.config.fileConfig(config_file)
-        except configparser.NoSectionError as e:  # pragma: no cover
-            print(e)
```

Comment:
I think we should keep the logging from .ini. For example, it allows to turn-on debug during migrations